_posts/en/pages/faq/2016-01-21-rbf-faq.md

```diff
@@ -0,0 +1,318 @@
+
+Because of these reasons and because the system behaving in expected
+ways is more secure and protective than the system behaving in
+"unpredictable but better on average ways", he proposed making
+replacement happen for all transactions. He also proposed a protocol
+to remove economic gains from double spending in a strong incentive
+compatible way, called replacement scorched earth, where if someone
+attempts to double spend you, you spend all the funds to fees so the
+attacker doesn't get them.  (But this is the kind of proposal only a
+game theorist could really love.)
+
+After Peter's proposal, we heard a number of wallets call very strongly
+for opt-in RBF, as they really needed a reasonable way to handle fees
+and saw opt-in RBF as a good way to do so which is why the proposal was
+changed to match Satoshi's original opt-in behavior.
+
+## Was the opt-in RBF pull request controversial?
+
+Not in the slightest. After extensive informal discussion stemming back
+months, the [PR][RBF PR] was opened on October 22nd. It was subsequently
+discussed in at least four Bitcoin development weekly meetings
+([2015-11-05][], [2015-11-12][], [2015-11-19][], and [2015-11-26][]).
+
+[RBF PR]: https://github.com/bitcoin/bitcoin/pull/6871
+[2015-11-05]: /en/meetings/2015/11/05/
+[2015-11-12]: /en/meetings/2015/11/12/
+[2015-11-19]: /en/meetings/2015/11/19/
+[2015-11-26]: /en/meetings/2015/11/26/
+
+In the PR discussion, 19 people commented (including people working on at least three different wallet brands) and 14 people explicitly ACKed the change, including at least one person who had been very outspoken in the past against full RBF.  No clearly negative feedback was provided in the PR (or elsewhere that we are aware of) while the PR was open.
+
+## When and how does this change get activated?
+
+Opt-in RBF doesn't really have an "activation" because it's not a part of Bitcoin's consensus. It's a local policy behavior and so the change happens one node at a time as people adopt software that behaves differently.  There are already some nodes on the network which perform RBF in various ways and there have been for some time, probably years.
+
+Bitcoin Core 0.12 will be released around February 2016 and will include opt-in RBF, and after that it will likely take a couple additional months before the behavior is commonplace. This may be sped up by the development of applications that make interesting use of it.
+
+## Is opt-in RBF only useful for adjusting fees?
+
+No, opt-in RBF can also be used to make slow-to-confirm transactions much more efficient by revising them to pay multiple people when the sender finds they have more people to pay and their last payment hasn't confirmed yet, or when they find they need to increase the payment amount for a person they are already paying. This also lets transaction authors avoid spending unconfirmed change, which is currently vulnerable to third-party malleability.
```

Comment:
How about this for the new paragraph in its entirety:

> No, another useful thing wallets can do with opt-in RBF is combine two or payments into a single payment when the first payment hasn't confirmed yet. This can save a large number of bytes and transaction fees even though the replacement will have pay a higher fee than the original.